frontend/lib/api/base/base-clients.ts

```diff
@@ -21,13 +21,14 @@ export abstract class BaseAPI {
 
 export abstract class BaseCRUDAPI<CreateType, ReadType, UpdateType = CreateType>
   extends BaseAPI
-  implements CrudAPIInterface {
+  implements CrudAPIInterface
+{
   abstract baseRoute: string;
   abstract itemRoute(itemId: string | number): string;
 
   async getAll(page = 1, perPage = -1, params = {} as any) {
     return await this.requests.get<PaginationData<ReadType>>(this.baseRoute, {
-      params: { page, perPage, ...params },
+      params: { page, perPage, orderBy: "position", orderDirection: "asc", ...params },
```

Comment:
Why implement this in the base class instead of the one where it applies? Most objects don't have a "position" field

Reply:
Uhhmmm... yeah, this is stupid :)
I might have gone a bit too fast with doing all the fixes yesterday, I will do this properly in a moment